src/common/stockitem.cpp

```diff
@@ -188,7 +188,7 @@ wxString wxGetStockLabel(wxWindowID id, long flags)
         STOCKITEM(wxID_REPLACE,             _("Rep&lace"),            _("Replace"));
         STOCKITEM(wxID_REVERT_TO_SAVED,     _("Revert to Saved"),     _("Revert to Saved"));
         STOCKITEM(wxID_SAVE,                _("&Save"),               _("Save"));
-        STOCKITEM(wxID_SAVEAS,              _("&Save as"),            _("Save as"));
+        STOCKITEM(wxID_SAVEAS,              _("Save &as..."),         _("Save as..."));
```

Comment:
While you're at it, could you capitalize the As? If you look at e.g. notepad.exe the As starts with a capital.

Reply:
I left it lower case because the Microsoft document I linked used lower case for "Save as...", however I concede that "Save As..." looks better to me.  I checked and Word, Notepad, and Visual Studio all use upper case 'A' so I will change it.

While we're at it, Notepad has "Find..." and "Replace..."

Reply:
AFAIR the trailing dots tell the user that other action will be required after choosing that option. It's more like a UI rule than a native or not thing, isn't it? Also, it has nothing to do with the modality of the dialog (see Notepad++), or even the presence of a dialog as is the case of Chrome.

As `Find` and `Replace` will probably always show a search box, they should most likely get the trailing dots, it's just the reason for it that I thought of pointing out.

Reply:
Sublime Text, Geany, NetBeans, gedit and LibreOffice all have these entries:
Find...
[Find and] Replace... 
Save As...